src/css/parse.js

```diff
@@ -2722,6 +2730,130 @@ function getUnary() {
 }
 
 /**
+ * Check if token is a unicode range (single or multiple <urange> nodes)
+ * @param {number} i Token's index
+ * @return {number} Unicode range node's length
+ */
+function checkUnicodeRange(i) {
+  const start = i;
+  let l;
+
+  if (i >= tokensLength) return 0;
+
+  if (l = checkUrange(i)) i += l;
+  else return 0;
+
+  while (i < tokensLength) {
+    if (l = checkSC(i)) i += l;
+    if (l = checkDelim(i)) i += l;
+    else break;
+
+    if (l = checkSC(i)) i += l;
+    if (l = checkUrange(i)) i += l;
+    else break;
+  }
```

Comment:
`i` should be modified only if another `urange` is found.
Consider a few examples:
```css
// A space after urange will be included into `unicodeRange` node.
unicode-range: U+416 ;

// Following two should throw parsing error, but instead they will
// now include spaces and comma as part of `unicodeRange` node.
unicode-range: U+416 ,;
unicode-range: U+416 , ;
```